ceph-defaults/defaults/main.yml

```diff
@@ -12,6 +12,7 @@ ceph_pkgs:
   - chrony
   - cephadm
   - podman
+  - ceph-common
```

Comment:
I don't think we can do that for downstream just yet.

Reply:
could you elaborate a bit more?

Reply:
Let me come back to you